client/reader/stream/reader-sidebar-followed-sites/index.jsx

```diff
@@ -111,34 +81,22 @@ export class ReaderSidebarFollowedSites extends Component {
 		}
 
 		return (
-			<ExpandableSidebarMenu
-				expanded={ this.props.isFollowingOpen }
-				title={ translate( 'Followed Sites' ) }
-				onClick={ this.props.toggleReaderSidebarFollowing }
-				materialIcon="check_circle"
-				disableFlyout={ true }
-				className={
-					( '/read' === path ||
-						sites.some(
-							( site ) =>
-								`/read/feeds/${ site.feed_ID }` === path || `/read/blogs/${ site.blog_ID }` === path
-						) ) &&
-					'sidebar__menu--selected'
-				}
-			>
-				{ this.renderAll() }
-				{ this.renderSites( sitesToShow ) }
-				{ ! allSitesLoaded && (
-					<Button
-						plain
-						// eslint-disable-next-line wpcalypso/jsx-classname-namespace
-						className="sidebar-streams__following-load-more"
-						onClick={ this.loadMoreSites }
-					>
-						{ translate( 'Load more sites' ) }
-					</Button>
-				) }
-			</ExpandableSidebarMenu>
+			<>
+				<h2>{ translate( 'Following' ) }</h2>
+				<ul>
+					{ this.renderSites( sitesToShow ) }
+					{ ! allSitesLoaded && (
+						<Button
```

Comment:
A button directly inside a `<ul />` element doesn't seem like semantic HTML.

Reply:
Doh! Good catch. Added an `li` wrapper in: https://github.com/Automattic/wp-calypso/pull/70667/commits/4b179788dbbf28f4ed63c3075d80367360eed3bf